Makefile

```diff
@@ -48,7 +48,8 @@ SRCS-y += lib/mailbox.c lib/net.c lib/flow.c lib/ipip.c \
 
 LDLIBS += $(LDIR) -Bstatic -lluajit-5.1 -Bdynamic -lm -lmnl
 CFLAGS += $(WERROR_FLAGS) -I${GATEKEEPER}/include -I/usr/local/include/luajit-2.0/
-EXTRA_CFLAGS += -O3 -g -Wfatal-errors -DALLOW_EXPERIMENTAL_API
+EXTRA_CFLAGS += -O3 -g -Wfatal-errors -DALLOW_EXPERIMENTAL_API \
+	-DLOG_RATELIMIT_INTERVAL=5 -DLOG_RATELIMIT_BURST=10
```

Comment:
1. These parameters should come from each block configuration like the level is currently set.

2. Blocks can now change their log level dynamically. Since the level is part of the data structure of the log rate, we need to change the Lua function that updates the log level to update both.

3. What are the values of these parameters that the kernel uses? How did you derive Gatekeeper values?

Reply:
3. The values are from the kernel [net_ratelimit()](https://github.com/AltraMayor/XIA-for-Linux/blob/337f3cfb501271ca3751f9ac690d2dd1951342d9/net/core/utils.c#L36), which is an overall limit. However, for gatekeeper, we will make each functional block configurable in terms of these values. As each functional block may require different logs, should we tag this with ``XXX`` for setting each block individually?

Reply:
The kernel uses `5 * HZ` instead of `5`, doesn't that mean 5s instead of 5ms?

Reply:
Good catch! In the kernel, it uses [jiffies](https://github.com/AltraMayor/XIA-for-Linux/blob/337f3cfb501271ca3751f9ac690d2dd1951342d9/lib/ratelimit.c#L46) to represent the intervals, which is defined using [HZ](https://github.com/AltraMayor/XIA-for-Linux/blob/337f3cfb501271ca3751f9ac690d2dd1951342d9/include/linux/jiffies.h#L167).